src/rest/handlers/rest_edges.erl

```diff
@@ -54,36 +60,19 @@ content_types_accepted(Req, State) ->
     {[{<<"application/json">>, from_json}], Req, State}.
 
 resource_exists(Req, State) ->
-    #{from := From, to := To} = State,
-    {ok, Result} = graph:edge_exists(From, To),
-    {Result, Req, State}.
-
-is_conflict(Req, State) ->
-    Result = case maps:get(operation, State) of
-                 add -> true;
-                 _ -> false
-             end,
-    {Result, Req, State}.
+    {false, Req, State}.
```

Comment:
rozumiem, że nic się nie zepsuje, jeśli to tak zostawimy?

Reply:
no ogólnie to przerzuciłem sprawdzanie warunków do from_json, a dalej do execute_operation

jeśli któryś z warunków jest niespełniony, to from_json zwróci {false, Req, State}, co sprawi, że klient otrzyma Status Code 400 Bad Request, wydaje mi się że to jest w porządku

Reply:
dobra, to już nie ma tematu

Reply:
mógłbym sprawdzać to tutaj, ale część kodu by się zduplikowała. jest to do rozważenia w sumie, ale na razie mi się spieszy, bo jeszcze mam (chyba ja :/ ) do ogarnięcia moduły graph i rest_basic_queries